src/test/utils/server_functions.js

```diff
@@ -135,6 +135,39 @@ async function create_system_and_check(server_ip) {
     }
 }
 
+
+//will create a system and check that the default account status is true.
+async function create_system(server_ip, port, protocol) {
```

Comment:
Should we remove create_system_and_check and clean_ova_and_create_system

Reply:
I think we can but some QA test still use them. @liranmauda?

Reply:
We can.
clean_ova is meant for getting the ova to a start point.
I think we should do it in a different PR so we will clean all the code from clean_ova